.gitignore

```diff
@@ -1,2 +1,3 @@
 *.pyc
 *.pyo
+rethinkdb_data/
```

Comment:
Is this actually going to be located in our project directory? I thought it was in ~/?
![screen shot 2015-09-22 at 2 21 34 pm](https://cloud.githubusercontent.com/assets/2533512/10030530/515e1194-6135-11e5-8d86-43717e3d1abb.png)